src/sgame/sg_active.cpp

```diff
@@ -34,6 +34,26 @@ bool ClientInactivityTimer( gentity_t *ent, bool active );
 static Cvar::Cvar<float> g_devolveReturnRate(
 	"g_devolveReturnRate", "Evolution points per second returned after devolving", Cvar::NONE, 0.4);
 
+static Cvar::Cvar<bool> g_poisonIgnoreArmor(
+	"g_poisonIgnoreArmor",
+	"Make poison damage armored and armorless humans equally",
+	Cvar::NONE,
+	false);
+static Cvar::Range<Cvar::Cvar<int>> g_poisonDamage(
+	"g_poisonDamage",
+	"Damage per second dealt to a poisoned individual",
+	Cvar::NONE,
+	5,
+	0,
+	500);
+static Cvar::Range<Cvar::Cvar<int>> g_poisonDuration(
```

Comment:
i think g_poisonDuration should have a higher resolution, despite currently only being checked every second.
it would be nice to define times in a coherent manner, in code all other settings are in milliseconds 
for cvars it could be enough to have it as a float

Reply:
I'd say I don't want to fix this problem because it would affect several things such as the "pain" reaction.

Should we fix that some day, then we can turn the cvar into a float that day.